test/index.html

```diff
@@ -9,11 +9,11 @@
     <link rel="stylesheet" href="//code.jquery.com/qunit/qunit-1.14.0.css">
     <script src="//code.jquery.com/qunit/qunit-1.14.0.js"></script>
     <!-- Load videojs-contrib-ads -->
-    <script src="../node_modules/videojs-contrib-ads/dist/videojs.ads.js"></script>
+    <script src="../node_modules/videojs-contrib-ads/dist/videojs-contrib-ads.js"></script>
     <!-- Load IMA library -->
     <script src="//s0.2mdn.net/instream/html5/ima3_debug.js"></script>
     <!-- Load local IMA plugin -->
-    <link rel="stylesheet" href="../src/videojs.ima.css">
+    <link rel="stylesheet" href="../src/videojs.ima.dev.css">
```

Comment:
is this intentional?

Reply:
It was, but this made me realize I don't have to do things this way anymore, so it's been reverted.